var/spack/repos/builtin/packages/helib/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Helib(CMakePackage):
+    """HElib is an open-source (Apache License v2.0) software library that
+    implements homomorphic encryption (HE). Currently available schemes are
+    the implementations of the Brakerski-Gentry-Vaikuntanathan (BGV) scheme
+    with bootstrapping and the Approximate Number scheme of Cheon-Kim-Kim-Song
+    (CKKS), along with many optimizations to make homomorphic evaluation run
+    faster, focusing mostly on effective use of the Smart-Vercauteren
+    ciphertext packing techniques and the Gentry-Halevi-Smart optimizations.
+    See this report for a description of a few of the algorithms using in this
+    library."""
```

Comment:
Can you have a short first paragraph, followed by a longer description later (disclaimer, I didn't check the snippet below for style) e.g.
```suggestion
    """HElib is an open-source (Apache License v2.0) software library that
    implements homomorphic encryption (HE). 
    
    Currently available schemes are the implementations of the 
    Brakerski-Gentry-Vaikuntanathan (BGV) scheme
    with bootstrapping and the Approximate Number scheme of Cheon-Kim-Kim-Song
    (CKKS), along with many optimizations to make homomorphic evaluation run
    faster, focusing mostly on effective use of the Smart-Vercauteren
    ciphertext packing techniques and the Gentry-Halevi-Smart optimizations.
    See this report for a description of a few of the algorithms using in this
    library."""
```
Spack will use the short paragraph in contexts where a short description is needed, and the full docstring where more info needs to be provided to users.